### PR TITLE
added functionality for get sitemap urls

### PR DESCRIPTION
**Pull request by t-chatoyan**

### CODE REVIEW

File: server/api/sitemap.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            data: response.results.map(page => ({
+                loc: `${url.pageName}${page.id}-${slugify(page.title)}`,
+            }))
+        };
+    };
+
+    const fetchAllPages = async () => {
+        const allData = [];
+
+        for (const url of urls) {
```
Comment (ridge-kimani) [inline server/api/sitemap.ts:31]: You could use `Promise.all` syntax.

Comment (t-chatoyan) [reply server/api/sitemap.ts]: Thanks for the review, added.

> You could use `Promise.all` syntax.

**Review by brian-mulier-p — CHANGES_REQUESTED** [review]
Move the dependency to the new one please :+1: 

And I typed everything in the API to remove any warnings, please in the future add types to prevent such warnings :+1:

File: nuxt.config.ts [hunk 2]
```diff
@@ -7,7 +7,9 @@ export default defineNuxtConfig({
       'nuxt-simple-sitemap',
```
[the remark this answers is not in the document]
Comment (t-chatoyan) [reply nuxt.config.ts:7]: sure, I saw it also, just used it because we had it in the project.

**Pull request merged**